app/src/main/java/com/infomaniak/mail/utils/SentryDebug.kt

```diff
@@ -104,11 +104,43 @@ object SentryDebug {
         )
     }
 
+    fun addAttachmentsBreadcrumb(draft: Draft) = with(draft) {
+
+        var count = 1
+        val data = mutableMapOf<String, Any>()
+
+        fun String.keyPad(): String = padStart(length = 15)
+        fun Int.countPad(): String = toString().padStart(length = 2, '0')
+        fun count(): String = "${count.countPad().also { count++ }}."
```

Comment:
`Pad` is a bit weird. I had to search what the padStart method did to understand what was the meaning of all this.

Reply:
What do you propose ?

Reply:
Fixed in #1988.